.github/actions/e2e-arc-test/action.yaml

```diff
@@ -0,0 +1,44 @@
+name: 'E2E ARC Test Action'
+description: 'Includes common arc installation, setup and test file run'
+
+inputs:
+  github-token:
+    description: 'A Github PAT'
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Helm installation
```

Comment:
our hosted runner doesn't have helm already installed?

Reply:
Seems it does, thanks!